android/app/src/main/res/drawable-v21/background_round_padded_pressed.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ripple xmlns:android="http://schemas.android.com/apk/res/android"
+    android:color="?attr/colorNavigationItemBackgroundPressed">
+    <item
+        android:id="@android:id/mask"
+        android:right="8dp">
```

Comment:
should `8dp`  be a global variable to be reused along with the other files?
Just ignore if Files from res folder are generated or are note "your code"

Reply:
This String is for testing, it will not exist, it was only for viewing on the screen